guides/source/debugging_rails_applications.md

```diff
@@ -337,7 +336,6 @@ Processing by ArticlesController#index as HTML
    10:     respond_to do |format|
    11:       format.html # index.html.erb
    12:       format.json { render json: @articles }
-
 (byebug)
```

Comment:
These are copy-pasted chunks of `byebug`'s output (I rerun them). Why not respect the original output?